tools/hangar/__snapshots__/test_corpus/sdk_tests/util/spawn.test.w_test_sim.md

```diff
@@ -0,0 +1,15 @@
+# [spawn.test.w](../../../../../../examples/tests/sdk_tests/util/spawn.test.w) | test | sim
+
+## stdout.log
+```log
+pass ┌ spawn.test.wsim » root/env0/test:spawn() with successful execution
+     │ Spawned child process with PID: 6962
```

Comment:
do we need to serialize the PID? or is it constant?

Reply:
PID is autogenerated afaik.

Reply:
so you'll have to serialize it on https://github.com/winglang/wing/blob/d4e17313d3c6e86b2737ec22b92764690cd66d0e/tools/hangar/src/utils.ts#L54-L55